### PR TITLE
Minor `VecDeque` doc examples cleanup.

### PR DESCRIPTION
**Pull request by frewsxcv**

### CODE REVIEW

**Comment by GuillaumeGomez** [comment]
Thanks!

@bors: r+ rollup

File: src/libcollections/vec_deque.rs [hunk 1]
```diff
@@ -726,18 +726,18 @@ impl<T> VecDeque<T> {
     /// ```
     /// use std::collections::VecDeque;
     ///
-    /// let mut vector: VecDeque<u32> = VecDeque::new();
+    /// let mut vector = VecDeque::<u32>::new();
```
Comment (bluss) [inline src/libcollections/vec_deque.rs]: The element type seems to not be important to the example. Even better to leave it out (and let it fall back to i32)?

File: src/libcollections/vec_deque.rs [hunk 2]
```diff
@@ -764,7 +764,7 @@ impl<T> VecDeque<T> {
...
```
Comment (bluss) [inline src/libcollections/vec_deque.rs]: same

**Review by bluss — CHANGES_REQUESTED** [review]

**Comment by GuillaumeGomez** [comment]
@bors: r-

@frewsxcv: Please fix @bluss's comments.

**Comment by frewsxcv** [comment]
Requested changes have been addressed.

**Review by bluss — APPROVED** [review]

**Pull request merged**